src/Administration/Resources/app/administration/src/module/sw-settings-services/component/sw-settings-services-deactivate-modal/sw-settings-services-deactivate-modal.spec.js

```diff
@@ -0,0 +1,94 @@
+import { mount } from '@vue/test-utils';
+import { MtModal, MtModalClose, MtModalAction } from '@shopware-ag/meteor-component-library';
+import SwSettingsServicesDeactivateModal from "./index";
+
+describe('src/module/sw-settings-services/component/sw-settings-services-deactivate-modal', () => {
+    const location = window.location;
+
+    beforeAll(() => {
+        Shopware.Service().register(
+            'shopwareServicesService',
+            () => ({
+                disableAllServices: jest.fn(),
+            })
+        )
+    });
+
+    beforeEach(() => {
+        Object.defineProperty(window, 'location', {
+            configurable: true,
+            value: { reload: jest.fn() },
+        });
+    })
+
+    afterEach(() => {
+        Object.defineProperty(window, 'location', { configurable: true, value: location });
+    })
+
+    it ('can be opened and closed', async () => {
+        const deactivateModal = await mount(SwSettingsServicesDeactivateModal);
+        await flushPromises();
```

Comment:
If I'm not mistaken this can be omitted? At least it succeeds even when removed 😄